src/lib/v2/mappers/local/sidebarTree.tsx

```diff
@@ -86,13 +86,16 @@ function getWorkspaceChildrenOrdered(
   sortingOrder: SidebarTreeSortingOrder,
```

Comment:
Please edit the `buildChildrenNavRows` function but keep in mind that there, items are not `LocalTreeItem` rather `SidebarTreeChildRow`, so I would actually like the compare function to:

1. Receive two `string` parameters and start prefix to compare them by
2. Compare the strings starts (items IDs start strings) and returns the results
3. The compare function be some utility function, possibly located in `src/lib/db/utils.ts` or `shared/lib/utils/array.ts` as there is where other sorting functions are (this will also be easy to share with cloud space which has the same problem)
4. The sort then called something like  `sortByAttributeAsc('label', rows).sort((a, b) => compareStringStart(a.id, b.id, FOLDER_ID_PREFIX))`
5. If the 4. step is a bit too complicated and it should be shared later, we can make this comparison function a global in `sidebarTree.tsx` and just send the IDs (and it can have hard-coded value for `FOLDER_ID_PREFIX`